src/update_notifier.rs

```diff
@@ -89,7 +89,9 @@ impl WapmUpdate {
                 }
 
                 let new_version = last_check.version.to_owned();
-                let old_version = util::get_latest_runtime_version()?;
+                // We use wasmer and not constants::DEFAULT_RUNTIME because the
+                // update logic is very tied to wasmer itself.
```

Comment:
Looks good!  When we start publishing binary releases on this repo, too, we can adjust this to look for wapm versions instead or look for both